Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "waffle"
-version = "0.0.44"
+version = "0.1.0"
 description = "Wasm Analysis Framework For Lightweight Experiments"
 authors = ["Chris Fallin <chris@cfallin.org>"]
 license = "Apache-2.0 WITH LLVM-exception"
```

Comment:
Can you also add `repository = "https://github.com/bytecodealliance/waffle"` here?

Reply:
Done!